index.md

```diff
@@ -7,4 +7,32 @@ position: 1
 tags: overview, welcome
 ---
 
-This is a placeholder document for the index page.
\ No newline at end of file
+The {{site.tp}} Knowledge Base portal gives answers to common user inquiries or issues, identified by the development team in the form of How-To and Troubleshooting articles. The easiest way to find relevant information is to search for key words in the *Search Knowledge Base* box at the top right corner.
```

Comment:
technically speaking the issues are identified by the support team or in more general way - the engineering team